src/pages/ct-purpose-exchanges/forms/PurposeOfExchangeForm.js

```diff
@@ -109,6 +110,23 @@ export default function PurposeOfExchangeForm({ labels, maxAccess, recordId, set
                 error={formik.touched.name && Boolean(formik.errors.name)}
               />
             </Grid>
+            <Grid item xs={12}>
+              <ResourceComboBox
+                endpointId={CurrencyTradingSettingsRepository.PurposeExchangeGroup.qry}
+                name='groupId'
+                label={labels.group}
+                valueField='recordId'
+                displayField='name'
+                values={formik.values}
+                required
```

Comment:
if the groupId is required add it to the validationSchema in the formik

Reply:
not required

Reply:
pusehed